vencloud: init at 0.0.1; nixos/vencloud: init (#374132)
NixOS/nixpkgs: nixos/modules/services/web-servers/vencloud.md

Reviewer: Aside from the suggestions here, I'm not sure if this should be included in general. Documentation is nice, but this doesn't really do more than link to the upstream docs -- which is already done in the option descriptions

Ideally this would look more like [Garage's](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/web-servers/garage.md) docs for example; otherwise I think it'd be fine to omit

Author: Well in the nixos modules checklist it says to add the docs which i did.

Reviewer: That isn't a hard requirement. If you want to keep it though, it will definitely need to be expanded, as in it's current form it's not super helpful IMO

Author: Well I will keep it incase things get deprecated or something drastically changes.

Reviewer: Then it should be extended. Again, in it's current form it's not very helpful; just keeping this around because *maybe* it'll be helpful one day doesn't make much sense

Deprecations should also not be hidden away in docs. These are instead usually spit out during evaluation time via assertions or option renaming

> # Vencord {#module-services-vencord}

Reviewer: ```suggestion
# Vencloud {#module-services-vencloud}
```

Author: Whoops I totally missed this, Thankyou!

> [Vencloud](https://github.com/vencord/vencloud) is a service that provides Cloud-Syncing to [vencord](https://github.com/Vendicated/Vencord).
> 
> 

Reviewer: ```suggestion

[Vencloud](https://github.com/vencord/vencloud) is a cloud settings sync API for [Vencord](https://github.com/Vendicated/Vencord).

```

> ## Basic Usage {#module-services-vencord-basic-usage}

Reviewer: ```suggestion
## Basic Usage {#module-services-vencloud-basic-usage}
```
This also needs to be updated for that to work

> VenCloud is exclusively configured through it's [settings](#opt-services.vencloud.settings) using enviroment virables as configuation. [Check out the enviroment example for more settings](https://github.com/Vencord/Vencloud/blob/main/.env.example).

Reviewer: ```suggestion
Vencloud is exclusively configured through it's [settings](#opt-services.vencloud.settings), using environment variables as configuration. [Check out the .env file example for more settings](https://github.com/Vencord/Vencloud/blob/main/.env.example).
```
